src/intrinsics.cpp

```diff
@@ -918,6 +920,180 @@ static Value *emit_intrinsic(intrinsic f, jl_value_t **args, size_t nargs,
     Value *den;
     Value *typemin;
     switch (f) {
+    HANDLE(bytevec_ref,2) {
```

Comment:
these should probably be builtin function rather than intrinsics, unless jeff has some other preference?